composer/algorithms/randaugment/README.md

```diff
@@ -0,0 +1,123 @@
+# 🎲 RandAugment
```

Comment:
@jfrankle RandAugment was already done in [docs/source/method_cards/randaugment.md](https://github.com/mosaicml/composer/pull/589/files#diff-f62ae31daf852cd5f974369e18a64471df521b103e43a0010682c839e7722a4f) below. This is a duplicate that file is just in the wrong folder right now.

Reply:
We're moving all of the method cards to the READMEs here. Those old method cards are now deprecated. This isn't a duplicate - it's an updated version.

Reply:
I moved the implementation and example code from the `docs/source/method_cards` version to the `readme` version